forge-game/src/main/java/forge/game/player/PlayerProperty.java

```diff
@@ -490,7 +490,9 @@ else if (kind.startsWith("Type")) {
                 }
             }
             return false;
-        }  
+        } else {
+            return false;
```

Comment:
Does this mean we're always returning false instead of returning true?

Reply:
Its also true for Card->CardState that does return false for Properties that doesn't exist

Reply:
I tried to think of a scenario where we'd want that but came up short